code/modules/reagents/chemistry/reagents/drinks.dm

```diff
@@ -193,6 +193,23 @@
 		holder.remove_reagent("capsaicin", 2)
 	..()
 
+/datum/reagent/consumable/drink/milk/on_skeleton_life(mob/living/carbon/human/H)
```

Comment:
`id == "milk"` doesn't let the children proc this code.
The other milk-type reagents have different ids from milk, and so will now proc this code due to inheritance.